### DISCUSSION TITLE
UV-K5 - UV-K58

### DISCUSSION THREAD

**Discussion by OE1MWW** [Tunas1337/UV-K5-Modded-Firmwares#48] [General]
are those two (UV-K5 and UV-K58) electrical / firmware the same ?
Any infos please?

**Comment by Tunas1337** [comment]
Yes, they are identical in terms of everything including the MCU. The difference is the amber display and the addition of a single capacitor to the audio path to improve the audio and cut harsh treble.

**Reply by RE3CON** [reply]
Will be great to know which capacitor is it part 526 or 528 or another and the value of it to change and to cut treble?
